services/app-api/types/reports.ts

```diff
@@ -9,11 +9,8 @@ export const isReportType = (x: string | undefined): x is ReportType => {
   return Object.values(ReportType).includes(x as ReportType);
 };
 
-export interface FormOptions {
-  type: ReportType;
-  stateOptions: string[];
-  state: string | undefined;
-  createdBy: string | undefined;
+export interface ReportOptions {
+  name: string;
```

Comment:
Pardon my ignorance here!
What's the difference between this interface and the one in ui-src/src/types/report.ts?
Do we ever share code across front and back end codebases (assuming that is the difference) ?

Reply:
There's no difference here other than ease of access! afaik we maintain one set of types in app-api and another in ui-src :) I did think about setting this up in just app-api and referencing it from there but the import reference was exceptionally long